python/setup.py

```diff
@@ -16,6 +16,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import print_function
```

Comment:
Eh, is it a mistake? This seems changing it to make it compatible with Python 2 and 3.

Reply:
Ah, you change it to be compatible on purpose. Ok, got it.

Reply:
Yes. We still need to set default python to python3 to build documentation: https://github.com/apache/spark/pull/26848#pullrequestreview-330516881

So we do not need to hardcode python3 in `dev/make-distribution.sh`